guides/v2.1/ui_comp_guide/concepts/ui_comp_linking_concept.md

```diff
@@ -147,7 +149,7 @@ Example of using `listens` in a component's configuration `.xml` file:
 <argument name="data" xsi:type="array">
     <item name="config" xsi:type="array">
         <item name="listens" xsi:type="array">
-            <item name="visible" xsi:type="string">sample_config.sample_provider:visibility</item>
+            <item name="sample_config.sample_provider:visibility" xsi:type="string">visibilityChanged</item>
```

Comment:
I think better to show listens functionality example like "local listeners". You listen for local property and handle local method. Because, previous example it's more imports and new(your) example inverted imports.

Reply:
The new example listens for changes in an external component  sample_config.sample_provider` and reacts locally via the callback method `visibilityChanged`. 

That is actually more like an import and also that is the most common use case I found as a third party developer for `listens`.

The previous example was very confusing to me, as it set the property `visibility` of the external component when the local property `visible` changes. That is more like an export.